webapi/webapi-contactsmanager-w3c-tests/contactsmanager/ContactsManager_save_Contact_fullset.html

```diff
@@ -68,7 +68,26 @@
     preferred: true,
     value: 'msn:kangx.xu@intel.com'
   });
-  var data = new Date(1987,10,27);
+  Date.prototype.format = function(format) {
```

Comment:
@zqzhang , I put my comment under the new PR. I think this PR should be closed now.